kitsune/search/v2/views.py

```diff
@@ -0,0 +1,114 @@
+import json
+
+from django.conf import settings
+from django.http import HttpResponse
+from django.utils.translation import ugettext as _, pgettext
+
+from kitsune import search as constants
+from kitsune.search.forms import SimpleSearchForm
+from kitsune.sumo.api_utils import JSONRenderer
+from kitsune.sumo.utils import build_paged_url
+from kitsune.products.models import Product
+from kitsune.search.views import _fallback_results
+from kitsune.search.utils import locale_or_default
+from kitsune.search.v2.search import Search, KBConfig, AAQConfig
+
+
+def simple_search(request):
+    search_form = SimpleSearchForm(request.GET, auto_id=False)
+
+    if not search_form.is_valid():
+        return HttpResponse(
+            json.dumps({"error": _("Invalid search data.")}),
+            content_type="application/json",
+            status=400,
+        )
+
+    cleaned = search_form.cleaned_data
+
+    # get language
+    language = locale_or_default(cleaned["language"] or request.LANGUAGE_CODE)
+    lang_name = settings.LANGUAGES_DICT.get(language.lower()) or ""
+
+    # get product
+    product = Product.objects.filter(slug__in=cleaned["product"]).first()
```

Comment:
Do we always have a single product here?

Reply:
why first?

Reply:
I think the API allows multiple products, however the way its currently used in instant search only allows for a single product (or all products) to be searched.